Sources/CLI/Image/ImageList.swift

```diff
@@ -37,16 +38,17 @@ extension Application {
         var global: Flags.Global
     }
 
-    struct ListImageImplementation {
-        static private func createHeader() -> [[String]] {
+    public struct ListImageImplementation {
```

Comment:
Sorry, just noticed this one. Do you need to make this public today for your plugin?

That we're hardwiring ListImageImplementation into ImageList is curious; I hadn't really looked at what was being done here before. Perhaps we just keep this stuff as is and come up with a better way later to achieve reusability for what's here.